src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/TidligereVedtaksperioderDto.kt

```diff
@@ -1,9 +1,29 @@
 package no.nav.familie.ef.sak.vilkår.dto
 
-data class TidligereVedtaksperioderDto(val infotrygd: TidligereInnvilgetVedtakDto?)
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.domene.TidligereInnvilgetVedtak
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.domene.TidligereVedtaksperioder
+
+data class TidligereVedtaksperioderDto(
+    val infotrygd: TidligereInnvilgetVedtakDto?,
+    val sak: TidligereInnvilgetVedtakDto?
+)
 
 data class TidligereInnvilgetVedtakDto(
     val harTidligereOvergangsstønad: Boolean,
     val harTidligereBarnetilsyn: Boolean,
     val harTidligereSkolepenger: Boolean
 )
+
+fun TidligereVedtaksperioder?.tilDto(): TidligereVedtaksperioderDto = this?.let {
+    TidligereVedtaksperioderDto(
+        infotrygd = it.infotrygd.tilDto(),
+        sak = it.sak?.tilDto()
+    )
+} ?: TidligereVedtaksperioderDto(null, null)
+
+fun TidligereInnvilgetVedtak.tilDto() =
+    TidligereInnvilgetVedtakDto(
+        harTidligereOvergangsstønad = this.harTidligereOvergangsstønad,
+        harTidligereBarnetilsyn = this.harTidligereBarnetilsyn,
```

Comment:
Viktig for frontend å skjønne forskjell på "null" og "false" i denne konteksten - "null" betyr ukjent og "false" betyr nei.

Reply:
harTidligere* er boolean, og i frontend har vi 
`export const mapTrueFalse = (bool: boolean): string => (bool ? 'Ja' : 'Nei');`

Hvis tidligereInnvilgetVedtak er null så vises ikke noe :)